backend/src/org/commcare/xml/EntryParser.java

```diff
@@ -30,7 +30,7 @@
 public class EntryParser extends CommCareElementParser<Entry> {
     private static final String FORM_ENTRY_TAG = "entry";
     private static final String VIEW_ENTRY_TAG = "view";
-    private static final String SYNC_REQUEST_TAG = "sync-request";
+    protected static final String SYNC_REQUEST_TAG = "remote-request";
```

Comment:
can you rename this variable to not include "sync", since nothing else does anymore